Practice Areas/Engineering/RFCs/API Code Isolation.md

```diff
@@ -0,0 +1,50 @@
+# RFC: Contractor code isolation and organization
```

Comment:
location: I don't love this "RFCs" folder, especially since the peer folders are "Backend", "Frontend", and "Infrastructure". maybe under "Backend/Notes", and date the file with YYYY-MM-DD for file ordering?

Reply:
Makes sense to me.  @kreek  will probably want to follow suit in his [API docs RFC PR](https://github.com/department-of-veterans-affairs/va.gov-team/pull/289)